doc/reference/core-builtin.md

```diff
@@ -1121,6 +1121,55 @@ Boolean and fold.
   lsts := lists
 ```
 
+### everymap
+``` scheme
+(everymap preds v [test: equal?]) -> boolean
+
+  preds := list of predicates or values
+  v     := value to compare with predicates
+  test  := optional, if preds contains a value, test is used for comparison
+```
+`everymap` returns `#t` if all predicates match. If `preds` contains a
+non-predicate, it is transformed into one.
```

Comment:
You need to be more explicit about the transformation. Also, about what is the default test.

Reply:
Also, in case the test predicate isn't symmetric, which is passed as first or second argument?